src/main/scala/io/iohk/ethereum/network/discovery/PeerDiscoveryManager.scala

```diff
@@ -35,16 +73,21 @@ class PeerDiscoveryManager(
         else
           knownNodesStorage.getKnownNodes().map(Node.fromUri)
 
-      bootstrapNodes ++ knownNodes
+      (bootstrapNodes ++ knownNodes).filterNot(isLocalNode).toVector
```

Comment:
We do, mainly because it will take nodes from the k-table as well which includes the local node. But the main motivation for putting it here as well was so it doesn't have to be done in the `sendRandomNodeInfo` as well, so that I can just pick a random index from it without having to repeat the exercise if it somehow happens to be the local node.